presto-parser/src/main/java/com/facebook/presto/sql/QueryUtil.java

```diff
@@ -181,6 +181,18 @@ public static Query simpleQuery(Select select, Relation from, Optional<Expressio
                 limit));
     }
 
+    public static Query singleValueQuery(String columnName, String value)
+    {
+        return query(new QuerySpecification(
```

Comment:
I think this can be simplified. See `StatementAnalyzer.visitExplain` which does the same thing